ramls/item.json

```diff
@@ -51,6 +51,10 @@
       "type": "string",
       "description": "Identifies the source of the call number, e.g., LCC, Dewey, NLM, etc."
     },
+    "effectiveCallNumber": {
+      "type": "string",
+      "description": "Effective Call Number is an identifier assigned to an item or its holding and associated with the item."
+    },
```

Comment:
As this is a derived property it should be marked as read only in the JSON schema, please update this definition